src/axiosCromwell.tsx

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const axiosCromwell = axios.create({
+  baseURL: 'http://localhost:3001/'
+});
```

Comment:
Nice! I'd add some other common things like `Content-Type` header (I guess you'll be communicating with server in JSON?) or CORS headers.

Reply:
You are definitely correct!
Done)